Tools/Scripts/compare-results

```diff
@@ -643,7 +703,18 @@ def main():
         if args.csv:
             writeCSV(speedometer2Breakdown(a), speedometer2Breakdown(b), args.csv)
 
-    elif typeA == MotionMark or typeA == MotionMark1_1 or typeA == MotionMark1_1_1:
+    elif typeA == Speedometer3:
+        if args.breakdown:
+            if args.detailed_breakdown:
+                dumpBreakdowns(speedometer3BreakdownSyncAsync(a), speedometer3BreakdownSyncAsync(b))
+            else:
+                dumpBreakdowns(speedometer3Breakdown(a), speedometer3Breakdown(b))
+
+        ttest(typeA, Speedometer3Results(a), Speedometer3Results(b))
+
+        if args.csv:
+            writeCSV(speedometer3Breakdown(a), speedometer3Breakdown(b), args.csv)
+    elif typeA == MotionMark or typeA == MotionMark1_1 or typeA == MotionMark1_1_1 or typeA == MotionMark1_2:
```

Comment:
Nit: this can be rewritten as:
```
elif any(typeA == validMotionMarkTest for validMotionMarkTest in [MotionMark, MotionMark1_1, MotionMark1_1_1, MotionMark1_2]):
```